### ISSUE TITLE
Elyra image unable to start due to missing oc-4 binaries under /opt/app-root/bin/

### ISSUE THREAD

**Issue by akchinSTC** [opendatahub-io/s2i-lab-elyra#5]
**Describe the bug**
Elyra image unable to start due to missing oc-4 binaries under /opt/app-root/bin/

**To Reproduce**

Openshift 4.3
Install opendatahub operator
created basic kfdef with jupyterhub definition only
log into spawner and select elyra image as starting image

**Expected behavior**
elyra should start up to launch page

**Screenshots**

**Additional context**
Error log on startup for notebook pod
```/opt/app-root/bin/oc: line 12: /opt/app-root/bin/oc-4: No such file or directory```

Elyra image is missing oc binaries as well as needing an updated oc symlink to the oc-wrapper script that is present in the s2i minimal notebook but not in the elyra image. 

lrwxrwxrwx 1 default root        31 Jul 23 15:08 oc -> /opt/app-root/bin/oc-wrapper.sh
-rwxrwxr-x 1 default root 121072360 Oct 10  2019 oc-3.11
-rwxrwxr-x 1 default root  76671080 May 23 03:18 oc-4.1
-rwxrwxr-x 1 default root  75280664 May  2 06:41 oc-4.2
-rwxrwxr-x 1 default root  84482424 May 23 10:00 oc-4.3

**Comment by akchinSTC** [comment]
@vpavlin - hello again, just tried launching elyra in a fresh openshift cluster and saw this error. Since the build config overlays  in odh-manifests are set to the s2i-minimal-notebook as a base image, are the binaries being removed somewhere during the build?

**Comment by akchinSTC** [comment]
looks like the s2i-elyra image is being based off the v0.0.5 tag of the s2i-minimal-notebook and not the v0.0.4 version, from looking at the manifest in quay.io

**Comment by vpavlin** [comment]
Hi @akchinSTC Yeah, it seems like this (https://github.com/opendatahub-io/odh-manifests/blob/master/jupyterhub/notebook-images/overlays/build/elyra-notebook-buildconfig.yaml#L20) needs to be updated.

Are you saying the released elyra image (v0.0.2) does not work or that the build config (which is basically v0.0.1) does not work?

**Comment by akchinSTC** [comment]
@vpavlin - The released image (v0.0.2) is not working at the moment because based on looking at the image layers, it is using a different image tag (v0.0.5, which is incorrect, it should be using v0.0.4)

![image](https://user-images.githubusercontent.com/13340013/94461186-a4550e00-016e-11eb-80fa-ca3e23a6790b.png)

**Comment by vpavlin** [comment]
@akchinSTC I tried elyra image in versions 0.0.2 and newly created 0.0.3 and I don't see this error. Elyra starts fine for me and when I check `oc` in terminal, it seems good

```
[1000570000@jupyterhub-nb-vpavlin ~]$ which oc
/opt/app-root/bin/oc
[1000570000@jupyterhub-nb-vpavlin ~]$ cat /opt/app-root/bin/oc
#!/bin/bash

case $OC_VERSION in
    4.*)
        OC_VERSION=4
        ;;
    *)
        OC_VERSION=3.11
        ;;
esac

exec /opt/app-root/bin/oc-$OC_VERSION "$@"
```

**Comment by vpavlin** [comment]
Seems like for some reason the file `oc-4` is actually missing from the base image. @harshad16 will look into it

**Comment by harshad16** [comment]
the file oc-4 was missing due to missing parts while download of binary.
it was fixed with this [commit](https://github.com/thoth-station/s2i-minimal-notebook/commit/3fb891ea30938a04e01ce510932d8d14c01485e9) in s2i-minimal-notebook. 
This would be available in a new image of s2i-minimal-notebook v0.0.6.
and would be updated for this repo as well.
